packages/webapp/src/components/LeftNavBar.tsx

```diff
@@ -154,7 +154,7 @@ export default function LeftNavBar(props: LeftNavBarProps) {
                         </div>
                     )}
                     <div className="space-y-1">
-                        {env === 'dev' && (
+                        {isCloud() && env === 'dev' && (
```

Comment:
same question here

Reply:
I didn't reviewed this sorry, but that means we can't access the getting started locally. It's probably okay but I'm not sure it was the goal :D

Reply:
No worries. It wasn't the goal but was necessary to properly lock the getting-started page down. The only reason it was so permissive before was to make it easy to work on locally. Locally already as it is we can't access the team page without editing some code in the `App.tsx` so I felt this was ok